contents/docs/self-host/migrate/migrate-from-amplitude.mdx

```diff
@@ -0,0 +1,82 @@
+# Migrate from Amplitude to PostHog
+
+If you're running close to Amplitude's 10 million free events limit, migrating to PostHog could be a good idea, especially since the PostHog event schema is quite similar to Amplitude's.
+
+## Steps
+
+1. Export your data from Amplitude using the [Amplitude Export API](https://developers.amplitude.com/docs/export-api)
+2. [Import data into PostHog](/docs/integrate/ingest-historic-data#importing-events) using any of PostHog's SDKs or capture API to 
```

Comment:
```suggestion
2. [Import data into PostHog](/docs/integrate/ingest-historic-data#importing-events) using any of PostHog's SDKs or capture API.
```

This sentence seems to just end at the moment.

Reply:
Whoops, you're right! My bad -- I'll commit the suggestion